src/neptune/internal/operation_processors/async_operation_processor.py

```diff
@@ -352,8 +350,6 @@ def process_batch(self, batch: List[Operation], version: int) -> None:
                     self._processor._last_ack = monotonic()
                     self._processor._lag_exceeded = False
 
-                    self._errors_processor.handle(errors)
```

Comment:
Shouldn't we restore the following error printing code?
```python
for error in errors:
    logger.error("Error occurred during asynchronous operation processing: %s", error,)
```

Reply:
Right, sorry